src/android/java/io/jxcore/node/JXcoreExtension.java

```diff
@@ -4,37 +4,30 @@
 
 import io.jxcore.node.jxcore.JXcoreCallback;
 import java.util.ArrayList;
-import android.annotation.SuppressLint;
-import android.content.Context;
-import android.graphics.Point;
-import android.provider.Settings.SettingNotFoundException;
 import android.util.Log;
-import android.view.Display;
-import android.view.WindowManager;
 import android.widget.Toast;
 
 import org.thaliproject.p2p.btconnectorlib.BTConnector;
 
 public class JXcoreExtension {
 
-    public static String EVENTSTRING_PEERAVAILABILITY   = "peerAvailabilityChanged";
-    public static String EVENTVALUESTRING_PEERID        = "peerIdentifier";
-    public static String EVENTVALUESTRING_PEERNAME      = "peerName";
-    public static String EVENTVALUESTRING_PEERAVAILABLE = "peerAvailable";
+    public final static String EVENTSTRING_PEERAVAILABILITY   = "peerAvailabilityChanged";
+    public final static String EVENTVALUESTRING_PEERID        = "peerIdentifier";
+    public final static String EVENTVALUESTRING_PEERNAME      = "peerName";
+    public final static String EVENTVALUESTRING_PEERAVAILABLE = "peerAvailable";
 
-    public static String EVENTSTRING_NETWORKCHANGED     = "networkChanged";
-    public static String EVENTVALUESTRING_REACHABLE     = "isReachable";
-    public static String EVENTVALUESTRING_WIFI          = "isWiFi";
+    public final static String EVENTSTRING_NETWORKCHANGED     = "networkChanged";
+    public final static String EVENTVALUESTRING_REACHABLE     = "isReachable";
+    public final static String EVENTVALUESTRING_WIFI          = "isWiFi";
 
-    public static String METHODSTRING_SHOWTOAST         = "ShowToast";
+    public final static String METHODSTRING_SHOWTOAST         = "ShowToast";
 
-    public static String METHODSTRING_STARTBROADCAST    = "StartBroadcasting";
-    public static String METHODSTRING_STOPBROADCAST     = "StopBroadcasting";
-
-    public static String METHODSTRING_CONNECTTOPEER     = "Connect";
-    public static String METHODSTRING_DISCONNECTPEER    = "Disconnect";
-    public static String METHODSTRING_KILLCONNECTION    = "KillConnection";
+    public final static String METHODSTRING_STARTBROADCAST    = "StartBroadcasting";
+    public final static String METHODSTRING_STOPBROADCAST     = "StopBroadcasting";
 
+    public final static String METHODSTRING_CONNECTTOPEER     = "Connect";
+    public final static String METHODSTRING_DISCONNECTPEER    = "Disconnect";
+    public final static String METHODSTRING_KILLCONNECTION    = "KillConnection";
```

Comment:
In line 45 below it is not necessarily to box (since it will just be unboxed). You can just write (Boolean) params.get(1)